step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/reports/AssertReportNode.java

```diff
@@ -28,7 +28,7 @@ public class AssertReportNode extends ReportNode {
 	
 	String key;
 	
-	String actual;
+	Object actual;

```

Comment:
@iegorov777 ReportNode belong to the model shared with the UI and REST services. Changing the type might impact the UI and Rest clients. These objects are also used in freemarker templates in the EE project step-reporting. I've checked in the UI and couldn't find usage of these fields. Rest clients might have to be adapted but if this change is required, we'll communicate it in the migration steps of the release

Reply:
@jeromecomte 
To ensure the compatibily I can revert the type back to String here.

Reply:
I've changed the 'actual' value to String.